BreanaMarie/w03-d2-data-mapper.md

```diff
@@ -0,0 +1,24 @@
+How would you model this data with JavaScript object types?
+
+	var cohort = (bestCohort, alsoBestCohort, hobby1, hobby2, hobby3);
+
+What attributes would you include on a Hobby object type? 
+	solor or group
+	legnth of time/commitment 
+
+What attributes would you include in a Cohort object type? What methods might you create for a Cohort object type to help us manage cohort data?	
+	teacher								.sort()
+	cohort name							.serialize()
```

Comment:
these methods are more about working with the page... we want to work with more abstract objects